packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -283,6 +295,33 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
     [$onModelGraftMutation](_event: Event) {
       this[$needsRender]();
     }
+
+    /** @export */
+    async exportScene(options: {
```

Comment:
Can we add documentation on this method to our index.html page of modelviewer.dev? You can put a link to three's docs page so you don't have to describe each option.